docs/framework/windows-workflow-foundation/how-to-enable-sql-persistence-for-workflows-and-workflow-services.md

```diff
@@ -3,7 +3,6 @@ title: "How to: Enable SQL Persistence for Workflows and Workflow Services"
 ms.date: "03/30/2017"
 dev_langs:
   - "csharp"
-  - "vb"
```

Comment:
Article doesn't have any vb code. So removed this.

Reply:
you can remove the whole thing then. csharp is the default one: https://github.com/dotnet/docs/blob/master/docfx.json#L97